backend/app/graphql/meal_request.py

```diff
@@ -17,20 +23,20 @@ class MealRequestTypeInput(graphene.InputObjectType):
 
 class MealTypeInput(graphene.InputObjectType):
     portions = graphene.Int(required=True)
-    dietary_restrictions = graphene.String(required=True)
-    meal_suggestions = graphene.String(required=True)
+    dietary_restrictions = graphene.String(default_value=None)
+    meal_suggestions = graphene.String(default_value=None)
 
 
 # Response Types
 class MealInfoResponse(graphene.ObjectType):
     portions = graphene.Int(required=True)
-    dietary_restrictions = graphene.String(required=True)
-    meal_suggestions = graphene.String(required=True)
+    dietary_restrictions = graphene.String()
+    meal_suggestions = graphene.String()
 
 
 class CreateMealRequestResponse(graphene.ObjectType):
     id = graphene.ID()
-    donation_datetime = graphene.DateTime(required=True)
+    drop_off_datetime = graphene.DateTime(required=True)
```

Comment:
Changed the name of this field to match the label used in the frontend ("drop-off time").